imports/api/blocks/server/methods.js

```diff
@@ -375,7 +375,10 @@ Meteor.methods({
                                             validator.profile_url =  getValidatorProfileUrl(validatorData.description.identity)
                                         validator.operator_address = validatorData.operator_address;
                                         validator.delegator_address = Meteor.call('getDelegator', validatorData.operator_address);
-                                        validator.jailed = validatorData.jailed;
+                                        
+                                        // validatorData.jailed may be undefined in cosmos-sdk v0.39.0.
+                                        validator.jailed = !!validatorData.jailed;
```

Comment:
@kwunyeung, this single line was the main problem I ran into after the `status` changes. `validatorData.jailed` is `undefined` instead of `false`.  This is a consequence of `ibc-alpha` moving to protobuf3 instead of Amino.

Reply:
That's actually a good move.